articles/azure-monitor/app/opentelemetry-nodejs-migrate.md

```diff
@@ -71,9 +71,9 @@ This guide provides two options to upgrade from the Azure Monitor Application In
 
 The following changes and limitations apply to both upgrade paths.
 
-##### Node < 14 support
+##### Node.js Version Support
 
-OpenTelemetry JavaScript's monitoring solutions officially support only Node version 14+. Check the [OpenTelemetry supported runtimes](https://github.com/open-telemetry/opentelemetry-js#supported-runtimes) for the latest updates. Users on older versions like Node 8, previously supported by the ApplicationInsights SDK, can still use OpenTelemetry solutions but can experience unexpected or breaking behavior.
+OpenTelemetry JavaScript's monitoring solutions officially support only Node version 14+. Check the [OpenTelemetry supported runtimes](https://github.com/open-telemetry/opentelemetry-js#supported-runtimes) for the latest updates. Users on older versions like Node 8, previously supported by the ApplicationInsights SDK, can still use OpenTelemetry solutions but can experience unexpected or breaking behavior. The ApplicationInsights SDK also depends on the Azure SDK for JS which does not guaruntee support for any Node.js versions that have reached end-of-life. See [the Azure SDK for JS support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md). For a version of Node.js to be supported by the ApplicationInsights 3.X SDK, it must have overlapping support from both the Azure SDK and OpenTelemetry.
```

Comment:
Is better to omit version 14+, this could change tomorrow and we will need to come to update here, only mention our package is dependent on what Otel supports

Reply:
Removed specific mention of Node 14.